src/main/kotlin/racingcar/Car.kt

```diff
@@ -1,16 +1,27 @@
 package racingcar
 
 class Car(
-    private var location: Int = 0,
+    private var _location: Int = DEFAULT_CAR_LOCATION,
+    val name: String,
 ) {
-    fun move(randNumber: Int) {
-        val rule = MovingJudgeRule.judge(randNumber)
-        val strategy = rule.strategy()
+    init {
+        require(isValidLength(name)) { "글자수는 5자를 넘을 수 없어요" }
+    }
+
+    val location
+        get() = _location
```

Comment:
[Kotlin Coding Convention](https://kotlinlang.org/docs/coding-conventions.html#class-layout)에 따르면 클래스는 아래의 순서대로 포맷을 지키라고 하고있어요! 😃

1. `Property declarations` and **initializer blocks**
2. Secondary constructors
3. Method declarations
4. Companion object

```suggestion
    val location
        get() = _location
        
    init {
        require(isValidLength(name)) { "글자수는 5자를 넘을 수 없어요" }
    }
```